runtime/astar/Cargo.toml

```diff
@@ -73,11 +73,11 @@ frame-system-benchmarking = { git = "https://github.com/paritytech/substrate", b
 pallet-collective = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false, optional = true }
 
 # Astar pallets
-pallet-block-reward = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.16", default-features = false }
-pallet-collator-selection = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.16", default-features = false }
-pallet-custom-signatures = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.16", default-features = false }
+pallet-block-reward = { git = "https://github.com/AstarNetwork/astar-frame", tag = "polkadot-v0.9.16", default-features = false }
```

Comment:
Let’s escape of using same names tags and branches, I guess we could use namever tags here

Reply:
Sure, I'll remove this completely if you already have the script ready.